configure

```diff
@@ -1339,7 +1351,7 @@ Optional Features:
   --enable-FEATURE[=ARG]  include FEATURE [ARG=yes]
   --enable-manager=ARG    memory managers: no, builtin, memwatch, dmalloc,
                           gcollect, bcheck (defaults to builtin)
-  --enable-packetver=ARG  Sets the PACKETVER define. (see src/common/mmo.hpp)
+  --enable-packetver=ARG  Sets the PACKETVER define. (see src/common/mmo.h)
```

Comment:
why has this been changed to mmo.h

i guess a typo? under rathena it is mmo.hpp

Reply:
Bucause it's a generated file and I'm already fixing it in #5731, if I fix it in both I'm gonna get a harder conflict to resolve. 😄

Reply:
makes sense 👍 😄